### PR TITLE
RCPS-155: User Management System: Create Documentation for User Model

### PR DESCRIPTION
**Pull request by katieann7**

### CODE REVIEW

File: src/main/java/com/user/management/appl/model/user/User.java [hunk 1]
```diff
@@ -6,23 +6,36 @@
...
 import java.sql.Timestamp;
 /**
- * This is the User Model.
+ * POJO class for User objects. This class represents the User information.
  * */
 public class User {
-    /**
-     * This is the User field.
-     * */
+    /** A unique user id of the user. */
```
Comment (RoCS2024) [inline src/main/java/com/user/management/appl/model/user/User.java:12]: remove documentation for private properties

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comment

**Comment by katieann7** [comment]
done po sir

**Review by RoCS2024 — APPROVED** [review]
all good

**Pull request closed without merging**